canonical/ubuntu.com
# [WD-13130] upgrade packages

## Done

- Upgraded "concurrently", "eslint-plugin-prettier", "intl-tel-input" packages in package.json
- Removed "puppeteer" and "exports-loader" packages from package.json since they are not used anymore
- Upgraded "feedgen" library in requirements.txt
- Updated "prepare-form-inputs.js" file to comply with the upgraded "intl-tel-input" package
- Removed /desktop/statistics page together with its styles and JS since it's not shown on the website anymore

## QA

- Open the demo instance and check if there are no errors on missing packages in the console
- Open any contact form that has a phone number field and check that it works properly and that it's included in the payload after submitting
- Open /desktop/statistics page and verify that it redirects to /desktop and no errors are shown in the console

## Fixes

- https://warthogs.atlassian.net/browse/WD-13130
- https://github.com/canonical/ubuntu.com/pull/12389
